controllers/mandataris.ts

```diff
@@ -166,12 +166,9 @@ async function findDecision(mandatarisId: string): Promise<string | null> {
       STATUS_CODE.BAD_REQUEST,
     );
   }
-  const decision = await findDecisionForMandataris({
-    type: 'uri',
-    value: mandatarisId,
-  } as Term);
+  const result = await findDecisionAndLinkForMandataris(mandatarisId);
```

Comment:
Still something going wrong here, this method expects a uri, but this is an id, ore the call above goes wrong, because that does require an id.

Reply:
I'm fixing this in another PR.